sklearn/experimental/tests/test_enable_hist_gradient_boosting.py

```diff
@@ -0,0 +1,103 @@
+import tempfile
+from subprocess import check_output, STDOUT, CalledProcessError
+import os
+import os.path as op
+import sys
+import textwrap
+from subprocess import TimeoutExpired
+
+
+TIMEOUT = 60
+
+
+def _make_cwd_env():
+    """Helper to prepare environment for the child processes"""
+    # This is a port from cloudpickle https://github.com/cloudpipe/cloudpickle
+    sklearn_repo_folder = op.normpath(
+        op.join(op.dirname(__file__), '../..'))
+    env = os.environ.copy()
+    pythonpath = "{src}{sep}tests{pathsep}{src}".format(
+        src=sklearn_repo_folder, sep=os.sep, pathsep=os.pathsep)
+    env['PYTHONPATH'] = pythonpath
+    return sklearn_repo_folder, env
+
+
+def assert_run_python_script(source_code, timeout=TIMEOUT):
+    """Utility to help check pickleability of objects defined in __main__
```

Comment:
This docstring should be updated, for instance: "Check assertions in an independent Python subprocess"